python/mxnet/numpy/multiarray.py

```diff
@@ -577,7 +577,7 @@ def __setitem__(self, key, value):
             if not isinstance(key, tuple) or len(key) != 0:
                 raise IndexError('scalar tensor can only accept `()` as index')
             if isinstance(value, numeric_types):
-                self.full(value)
+                self._full(value)
```

Comment:
I think we should delete the class method `_full` from `ndarray` since NumPy ndarray class does not have it. Wherever it's used, we should replace it with the direct `full` operator call.

Reply:
Sure

Reply:
turns out `_full` is also used in `python/mxnet/ndarray/ndarray.py` for indexing, so keeping it for now.